Runtime/AvatarCreator/Data/AssetLibrary.cs

```diff
@@ -17,9 +17,7 @@ public struct PartnerAsset
         [JsonConverter(typeof(GenderConverter))]
         public OutfitGender Gender;
         [JsonProperty("iconUrl")]
-        public string Icon;
-        [JsonProperty("maskUrl")]
-        public string Mask;
+        public string ImageUrl;
```

Comment:
Maybe we should move this PartnerAsset to a separate script, what do you think?

Reply:
Also I suppose this is breaking stuff in current avatar creator?

Reply:
So everything will (or should) still work. But test it out and let me know if it doesn't. The icons have been updated now so that they are already masked basically.

Reply:
and yep we can move to a separate script

Reply:
I am also generally in favour of having a script/file per object declaration, normally in a 'models' folder but not sure if that naming would be as common in the Unity world. We should stay consistent though, so we should commit to doing this everywhere, or nowhere IMO.

Also, regarding it being a breaking change, even if it is, it shouldn't be an issue as eventually this (and a re-creation of the avatar creator using these) will replace the existing one. However, I guess any breaking changes should be merged to a feature branch rather than develop for that thought process to work?